App/Containers/ThreadsList.js

```diff
@@ -132,40 +132,6 @@ class ThreadsList extends React.PureComponent {
     )
   }
 
-  _onPressItem = (photo) => {
-    const { id } = photo
-    this.props.viewThread(id)
-    this.props.navigation.navigate('ViewThread')
-  }
-
-  _onRefresh = () => {
-    this.props.refreshMessages()
-  }
-
-  _keyExtractor = (item, index) => item.id
-
-  _renderItem = ({ item }) => {
-    return (
-      <ThreadCard id={item.id} {...item} profile={this.props.profile} onPress={this._onPressItem} />
-    )
-  }
-
-  _renderList () {
-    return (
-      // FIXME: This should be a FlatList for sure
-      <View style={styles.contentContainer} >
-        <FlatList
-          data={this.props.threads}
-          keyExtractor={this._keyExtractor}
-          renderItem={this._renderItem}
-          refreshing={this.props.refreshing}
-          onRefresh={this._onRefresh}
-          initialNumToRender={4}
-        />
-      </View>
-    )
-  }
-
```

Comment:
Really not much left in this class. Seems like you just moved 90% of the code to another file. Maybe I'm missing something.

Reply:
I guess navigation and placeholder stuff is handled in here... ok ok